api/tests/unit/application/certifications/certification-controller_test.js

```diff
@@ -139,27 +138,60 @@ describe('Unit | Controller | certifications-controller', () => {
   });
 
   describe('#getCertificationByVerificationCode', () => {
-    const certification = domainBuilder.buildPrivateCertificateWithCompetenceTree();
-    const serializedCertification = '{JSON}';
-
-    const verificationCode = 'P-123456BB';
-    const request = { payload: { verificationCode } };
-
-    beforeEach(() => {
-      sinon.stub(usecases, 'getShareableCertificate');
-      sinon.stub(certificationSerializer, 'serializeForSharing');
-    });
 
-    it('should return a serialized certification when use case returns a certification', async () => {
+    it('should return a serialized shareable certificate found in the usecase', async () => {
```

Comment:
je pense que "found in the usecase" n'est pas très utile comme précision et révèle un détail d'implem qui rend ce titre "fragile" (ex. demain si je change l'archi et que je n'ai pas de concept de usecase mais par exemple de queyHandler)